src/scss/components/masthead/_masthead.scss

```diff
@@ -14,7 +14,7 @@
 
         // Reduce the spacing below lockup
```

Comment:
You might want to change the 'reduce' to 'add' since it adds space.

Reply:
Fixed in the next pr.

Reply:
Fixed in the next pr.